group_project_v2/public/js/components/review_subject_selector.js

```diff
@@ -0,0 +1,63 @@
+/* exported ReviewSubjectSelectorXBlock, ReviewSubjectSelectorConstants */
+var ReviewSubjectSelectorConstants = {
+    status_icon_class: "group-project-review-state",
+    refresh_statuses_event: "group_project_v2.review.refresh_status"
+};
+
+function ReviewSubjectSelectorXBlock(runtime, element) {
+    "use strict";
+    // Set up gettext in case it isn't available in the client runtime:
+    if (typeof gettext === "undefined") {
+        window.gettext = function gettext_stub(string) { return string; };
+    }
+
+    var ERROR_REFRESHING_STATUSES = gettext("Error refreshing statuses");
+
+    var get_statuses_endpoint = runtime.handlerUrl(element, "get_statuses");
+    var status_icon_class = ReviewSubjectSelectorConstants.status_icon_class;
+
+    var group_project_dom = $(element).parents(".group-project-xblock-wrapper");
```

Comment:
This variable name sounds like this represents a DOM object, while it actually is a jQuery object.